google-cloud-clients/google-cloud-spanner/pom.xml

```diff
@@ -18,9 +18,29 @@
   </parent>
   <properties>
     <site.installationModule>google-cloud-spanner</site.installationModule>
+    <jacoco.skip>true</jacoco.skip>
   </properties>
   <build>
     <plugins>
+      <plugin>
+        <groupId>org.jacoco</groupId>
+        <artifactId>jacoco-maven-plugin</artifactId>
+        <version>0.8.2</version>
```

Comment:
check if google-cloud-bom lists this too. We may get the latest version from there, if it exists

Reply:
Do you mean in the bom's pom.xml? No, there is no jacoco in the entire repo.

Reply:
@pongad Michael, we have a question whether this should go in the bom rather than here. And whether jacoco should be added repo-wide (skipped by default) rather than just for spanner.

What do you suggest?

Reply:
I think here is fine. BOM is imported by other people, so I think test deps should go here.